### PR TITLE
Test with php 7.3

### PR DESCRIPTION
**Pull request by DeepDiver1975**

### CODE REVIEW

**Comment by DeepDiver1975** [comment]
dav is ready for php 7.3

next:
- merge
- tag alpha

File: lib/CalDAV/ICSExportPlugin.php [hunk 1]
```diff
@@ -125,7 +128,7 @@ function httpGet(RequestInterface $request, ResponseInterface $response) {
             $componentType = 'VEVENT';
         }
         if (isset($queryParams['componentType'])) {
-            if (!in_array($queryParams['componentType'], ['VEVENT', 'VTODO', 'VJOURNAL'])) {
+            if (!\in_array($queryParams['componentType'], ['VEVENT', 'VTODO', 'VJOURNAL'])) {
```
Comment (staabm) [inline lib/CalDAV/ICSExportPlugin.php]: Unrelated change?

File: lib/CalDAV/ICSExportPlugin.php [hunk 2]
```diff
@@ -318,7 +323,9 @@ function mergeObjects(array $properties, array $inputObjects) {
                     // VTIMEZONE is special, because we need to filter out the duplicates
                     case 'VTIMEZONE' :
                         // Naively just checking tzid.
-                        if (in_array((string)$child->TZID, $collectedTimezones)) continue;
+                        if (\in_array((string)$child->TZID, $collectedTimezones)) {
```
Comment (staabm) [inline lib/CalDAV/ICSExportPlugin.php]: Unrelated?

**Comment by DeepDiver1975** [comment]
Will Tag a version for composer tomorrow
